src/components/ComboBox/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface ComboBoxOption {
+  /**
+   * The label or display text
+   */
+  label?: string;
+  /**
+   * The option value (used for selection)
+   */
+  value: string;
+  /**
```

Comment:
probably should add a comment here explaining that this is for usecases where the display value inside the input after selecting an option doesn't exactly match what the option looked like in the dropdown.